Pirat/Model/Entity/OfferEntity.cs

```diff
@@ -29,6 +29,8 @@ public class OfferEntity : ProviderBase, Findable
 
         public int[] personal_ids { get; set; }
 
+        public DateTime timestamp { get; set; }
```

Comment:
Kann der timestamp vom Typ DateTime so in Postgres übernommen werden? In Postgres speichern wir den Timestamp als timestamp.

Reply:
Zumindest wird der Timestamp korrekt in der Datenbank abgespeichert. Deswegen bin ich davon ausgegangen, dass das Speichern an sich auch richtig funktioniert.